src/command_modules/azure-cli-advisor/azure/cli/command_modules/advisor/_help.py

```diff
@@ -6,54 +6,66 @@
 
 from knack.help_files import helps
 
-helps['advisor'] = """
-    type: group
-    short-summary: Manage Azure Advisor.
+helps["advisor configuration update"] = """
+"type": |-
+    command
+"short-summary": |-
+    Update Azure Advisor configuration.
 """
 
-helps['advisor configuration'] = """
-    type: group
-    short-summary: Manage Azure Advisor configuration.
+helps["advisor configuration"] = """
+"type": |-
```

Comment:
These syntactical changes are not required/desired. They make it seem like everything have changed when that isn't the case.